src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
You can't scroll anywhere near that fast in practice, but your point is taken.

If we're concerned about generating garbage here, I'll write the grungy logic to figure out which row was clicked and which object it corresponds to, and use that instead.